src/wazuh_db/wdb.h

```diff
@@ -2066,6 +2067,14 @@ wdbc_result wdb_global_unassign_agent_group(wdb_t *wdb, int id, cJSON* j_groups)
  */
 int wdb_global_groups_number_get(wdb_t *wdb, int agent_id);
 
+/**
+ * @brief Verifies that the group name complies with a predefined pattern.
```

Comment:
I would say `satisfies` instead of `complies`. Check https://wikidiff.com/comply/satisfy#:~:text=As%20verbs%20the%20difference%20between,self%3B%20to%20consent%20or%20conform.

Reply:
Changed